src/components/PopoverWithoutOverlay/types.ts

```diff
@@ -0,0 +1,28 @@
+import {View} from 'react-native';
+import BaseModalProps from '@components/Modal/types';
+import ChildrenProps from '@src/types/utils/ChildrenProps';
+
+type PopoverWithoutOverlayProps = ChildrenProps &
+    Omit<BaseModalProps, 'type' | 'popoverAnchorPosition'> & {
+        /** The anchor position of the popover */
+        anchorPosition?: {
+            top?: number;
+            right?: number;
+            bottom?: number;
+            left?: number;
+        };
+
+        /** The anchor ref of the popover */
+        anchorRef: React.RefObject<HTMLElement>;
+
+        /** A react-native-animatable animation timing for the modal display animation. */
+        animationInTiming?: number;
+
+        /** Whether disable the animations */
+        disableAnimation?: boolean;
+
+        /** The ref of the popover */
+        withoutOverlayRef: React.RefObject<HTMLElement & View>;
```

Comment:
Any issue with this suggestion?
```suggestion
        withoutOverlayRef?: React.RefObject<HTMLElement>;
```

Reply:
@situchan I've found only one `PopoverWithoutOverlay` component [usage](https://github.com/Expensify/App/blob/main/src/components/Popover/index.tsx#L82) and it's not optional. If we get rid of `View` in `withoutOverlayRef` typing we will have TS issue since this ref is then [passed](https://github.com/Expensify/App/pull/32171/files/5c0950d73596a7cda7d50e95fb95b715aabb7e3f#diff-9a09cbff47e501863332fa91b03e246243b8545a834886cabaeadcf8731551acR121) to the View. What do you think?

Reply:
ok and what about `& View` part?
No issue itself but just asking because it's inconsistent with other ref typings

Reply:
@situchan As I mentioned, if we get rid of `& View` part we will have TS issue in the place where we pass `withoutOverlayRef` to the [View ref](https://github.com/Expensify/App/pull/32171/files/5c0950d73596a7cda7d50e95fb95b715aabb7e3f#diff-9a09cbff47e501863332fa91b03e246243b8545a834886cabaeadcf8731551acR121).

We also have similar cases in the code: [one](https://github.com/Expensify/App/blob/main/src/components/SignInPageForm/index.tsx#L13), [two](https://github.com/Expensify/App/blob/main/src/components/Composer/index.tsx#L82)

Reply:
ok no problem